.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,26 @@
+# This runs the integration tests for kuberpult
+name: "Kuberpult Integration tests"
+'on':
+  pull_request:
+    branches:
```

Comment:
this means run it only on a branch called `pr` remove it

Reply:
Thank you!!! :)